.github/workflows/continuous-integration.yml

```diff
@@ -88,6 +88,21 @@ jobs:
           fi
           echo $HDF5_DIR
 
+      - name: Download a specific release of PETSc
+        run: |
+          PETSC_VERSION="v3.20.5"
+          wget https://gitlab.com/petsc/petsc/-/archive/$PETSC_VERSION/petsc-$PETSC_VERSION.zip
+          unzip petsc-$PETSC_VERSION.zip
+          rm petsc-$PETSC_VERSION.zip
+          mv petsc-$PETSC_VERSION petsc
+
+      - name: Install PETSc and petsc4py with complex support, and test it
+        working-directory: ./petsc
+        run: |
+          ./configure --with-scalar-type=complex --with-fortran-bindings=0 --have-numpy=1 --with-petsc4py=1 --with-64-bit-indices=1
+          make all check
+          python3 -c "import sys, os; sys.path.append(os.getcwd() + '/arch-linux-c-debug/lib')"
```

Comment:
`arch-linux` will not work on macOS

Reply:
Modifying `sys.path` here is local to the current Python process, which ends after the call. It will have no effect on future calls to Python.